package.json

```diff
@@ -20,6 +20,7 @@
     "tsc:blade": "yarn workspace @razorpay/blade typecheck",
     "lint": "eslint --ext js,jsx,ts,tsx packages",
     "release": "changeset publish",
+    "release-npm": "lerna run --scope @razorpay/blade npm-publish",
```

Comment:
can't we use changeset publish to publish to npm? or that is the iteration you'll do after testing?

Reply:
Can try it once after testing but `changeset publish` command is very limited so it doesn't support specifying registry with flag, dry runs, etc. So we might have to create a new npmrc and rename/copy-paste it which might get a bit tricky. Eventually I can make this command part of `release` command itself so we won't be calling this separately